src/pages/costModels/costModel/index.tsx

```diff
@@ -126,10 +128,24 @@ class CostModelInformation extends React.Component<Props, State> {
                   <PriceListTable costModel={current.name} assignees={sources.map(p => p.name)} current={current} />
                 </div>
               </TabContent>
-              <TabContent eventKey={1} id="refMarkup" ref={this.tabRefs[1]} hidden={this.state.tabIndex !== 1}>
-                <MarkupCard current={current} />
+              <TabContent
+                eventKey={1}
+                id="refCostCalculations"
+                ref={this.tabRefs[1]}
+                hidden={this.state.tabIndex !== 1}
+              >
+                <div style={styles.costCalculation}>
+                  <Grid hasGutter>
+                    <GridItem lg={6} id="refMarkup">
+                      <MarkupCard current={current} />
+                    </GridItem>
+                    <GridItem lg={6} id="refDistribution">
+                      <DistributionCard current={current} />
+                    </GridItem>
+                  </Grid>
```

Comment:
We need to hide the distribution card when the source isn't OpenShift.

In addition, I expect you will have a different grid layout to show markup only. For example, the current markup card spans the entire page width.

Reply:
It is hidden, it will not show up unless it is Openshift, and yes ewhen it is not open shift the markup card goes across the entire page. I'll add a screen shot